src/deployment/azuredeploy.json

```diff
@@ -101,6 +102,35 @@
             "apiVersion": "2018-11-30",
             "location": "[resourceGroup().location]"
         },
+        {
+            "type": "Microsoft.KeyVault/vaults",
+            "apiVersion": "2016-10-01",
```

Comment:
This seems to be a few versions behind the latest. Is there a specific reason we're using this version?

Reply:
updated to use the latest